salt/salt-auto-update.py

```diff
@@ -77,16 +72,20 @@ def load_file_lines(filename):
         return set()
 
 
-def match_start_event(event):
+def match_minion_ping(event):
     if event is None:
         return None
 
     tag = event.get("tag", "")
     if tag == "salt/event/exit":
         sys.exit()
 
-    return START_EVENT_PATTERN.match(tag)
-
+    if tag == "minion_ping":
+        data = event.get("data")
+        if data == None:
```

Comment:
`if data is None` is more correct (`None` is a singleton)

`if not data:` is even more idiomatic.